src/MLJFlux.jl

```diff
@@ -31,8 +33,7 @@ include("mlj_model_interface.jl")
 export NeuralNetworkRegressor, MultitargetNeuralNetworkRegressor
 export NeuralNetworkClassifier, NeuralNetworkBinaryClassifier, ImageClassifier
 export CUDALibs, CPU1
+export EntityEmbedder
 
 include("deprecated.jl")
-
-
-end #module
+end
```

Comment:
It's pretty standard practice to annotate a module's end this way:

```suggestion
end # module
```